template/2-history.md

```diff
@@ -2,42 +2,6 @@
-
-*Epic 2*
-- messages screen shows all chat histories that a user has with friends, and the friends screen shows a list of all the friends a user has connected with
-- once a user clicks on a message item in the messages list, they will see the full private chat history they have with a friend
-- once a user clicks on a friend list item, they will see the friend's profile
-- every user has a profile that they can edit in the profile screen
-- on signup, every user is given a unique QR code that they can use to access events and connect with friends
-
-*Epic 3*
-- the hosting screen is the dashboard for all event hosts
-- it features a list of all their active events which, when clicked, leads to a screen to edit those details
-- it allows hosts to add events easily
-- it also includes a map view allowing hosts to visualize the locations of their events
-- hosts get access to a tool that allows them to easily scan the QR codes of event goers and see if the event goers have signed up or not
-
-*What did you learn?*
-- many members of our team were new to the scrum process, so it was an adjustment period, especially at the beginning, to get used to agile development
-- often at the beginning we were so eager to correct issues with the code that we took on gigantic PRs that touched too many files, we learned from this mistake by 1) limiting the scope of PRs or splitting PRs if needed, and 2) writing better tasks in the SCRUM board that made the scope of the task clear to the developer
-- with an idea as ambitious as ours and a limited time to complete everything, we knew we would not be able to finish everything in time, so we had to make a lot of hard decisions regarding prioritization of tasks
-- we learned to break down tasks in a way that reduces dependencies between them
-
-*What is missing to bridge from PoC to MVP?*
-- there are still some user flows that we designed that are missing from the PoC
-- some important defining features regarding connecting with others still need to be implemented
-- some more user flows have to be thought through and designed to make sure they are the best for the user, such as group chats, in event geo-location, and history of attended events
+Throughout the development of the PoC, our team faced several learning curves, particularly with the Scrum process. Initially, we encountered challenges with managing large pull requests (PRs) and task clarity, which we addressed by refining task definitions on the Scrum board and limiting the PR scope, which also led to fewer dependencies between tasks. Furthermore, the ambitious nature of the app required us to make difficult prioritization decisions due to limited time. Despite these challenges, we successfully laid the groundwork for the MVP, although further development is needed to implement additional user flows and defining features such as group chats, in-event geo-location, and event history tracking to fully bridge from PoC to MVP.
```

Comment:
What is missing to bridge from PoC to MVP?

Suggestion: split into two paragraphs to make it clear we're answering questions.
```suggestion
Throughout the development of the PoC, our team faced several learning curves, particularly with the Scrum process. Initially, we encountered challenges with managing large pull requests (PRs) and task clarity, which we addressed by refining task definitions on the Scrum board and limiting the PR scope, which also led to fewer dependencies between tasks. Furthermore, the ambitious nature of the app required us to make difficult prioritization decisions due to limited time. 

Despite these challenges, we successfully laid the groundwork for the MVP. Further development is needed to implement additional user flows and define more core features that add value to the user, such as group chats, associating new connections to events, defining privacy levels for individual profiles, event photo sharing, in-event geo-location, and event history tracking to fully bridge from PoC to MVP.
```